### PR TITLE
Exercise 4

### PR DESCRIPTION
**Pull request by cristi-neagu**
Everything should be working as expected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,4 +63,12 @@ Simplest method of running tests is through VS Code, by using the `Test explorer
...
\ No newline at end of file
+```
+
+##  Provision a VM from an Ansible Control Node
+- Create folder `opt` in the home directory
+- Create folder `todoapp` in the `opt` folder
+- Install `git` using `sudo yum install git`
+- Clone the app repository in `~/opt/todoapp/`
+- Checkout branch `Exercise-4` with `git checkout Exercise-4`
+- Run the ansible playbook by using `ansible-playbook playbook -i ansible_inv`
```
Comment (JackMeadDev) [inline README.md:74]: Thanks for the clear instructions here!

File: playbook [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    become: yes
+    ansible.builtin.yum:
+      name: git
+      state: present
+  
+  - name: Install Python
+    become: yes
+    ansible.builtin.yum:
+      name: python3
+      state: present
```
Comment (JackMeadDev) [inline playbook:26]: Worth noting that these can be combined like so:

```suggestion
  - name: Install Git & Python
    become: yes
    ansible.builtin.yum:
      name: 
        - git
        - python3
      state: present
```

Up to you whether you think that's easier or harder to read though

File: playbook [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  - name: Install Poetry
+    ansible.builtin.shell:
+      cmd: curl -sSL https://install.python-poetry.org | python3
+      creates: ~/.local/bin/poetry
+
+  - name: Create To-Do App directory
+    ansible.builtin.shell:
+      cmd: mkdir -p ~/opt/todoapp
+      creates: ~/opt/todoapp
```
Comment (JackMeadDev) [inline playbook:36]: This works nicely, but worth noting there is a built in option for this:
```suggestion
    ansible.builtin.file:
      path: ~/opt/todoapp
      state: directory
```

**Review by JackMeadDev — APPROVED** [review]
Nice work, you've got a clean effective playbook and I can see it's running happily! I've left a couple of optional thoughts below, but very happy for you to merge this when you're ready 😄

Note that we won't be using these VMs again, so I'll schedule these to be torn down by the end of this week - let me know if that's an issue for any reason

File: playbook (hunk 2 above)
Comment (cristi-neagu) [reply playbook:26]: It is, but I think I prefer the individual steps.

File: playbook (hunk 3 above)
Comment (cristi-neagu) [reply playbook:36]: Good suggestion, thank you!

**Pull request merged**